plugins/woocommerce-admin/client/two-column-tasks/task-list-item.tsx

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
Thanks for creating a separate component for this, this is indeed much nicer and support the SlotFills.
I end up copying this file pretty well fully in a slightly simplified PR: #32742